scripts/git_divergence/divergence.py

```diff
@@ -5,7 +5,7 @@
 
 from git import Repo
 
-MASTER = "master"
+MASTER = "master" if os.getenv("GIT_AUTH_BRANCH") is None else os.getenv("GIT_AUTH_BRANCH")
```

Comment:
This might be easier to do something like: `MASTER = os.getenv("GIT_AUTH_BRANCH", "master")` :slightly_smiling_face:

Reply:
Oooo, I didn't know you can do that with Python; I thought I was trying to be smart, but I like that. 

https://docs.python.org/3.8/library/os.html#os.getenv Nice!